src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] defaultFigures = figureSupplier.getDefaultFigure();
+        Figure[] randomFigures = figureSupplier.getRandomFigure();
```

Comment:
Similarly, the method name `getRandomFigure` should be `getRandomFigures` if it is intended to return an array of figures.